drivers/gles3/storage/texture_storage.cpp

```diff
@@ -2119,7 +2119,7 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 				texture->layers = 1;
 			}
 			texture->gl_format_cache = rt->color_format;
-			texture->gl_type_cache = GL_UNSIGNED_BYTE;
+			texture->gl_type_cache = rt->hdr ? GL_HALF_FLOAT : GL_UNSIGNED_BYTE;
```

Comment:
This has a merge conflict now. The current line in `master` is:

```cpp
texture->gl_type_cache = !rt->hdr ? GL_UNSIGNED_BYTE : GL_FLOAT; // to set HDR format size to 8 and keep 4 for LDR format
```

How should this be handled?